packages/fast-usdc-contract/test/contract-setup.ts

```diff
@@ -432,43 +433,45 @@ export const makeCustomer = (
       await eventLoopIteration();
       return tx;
     },
+
     checkSent: (
       t: ExecutionContext<FucContext>,
       { bank = [] as any[], local = [] as any[] } = {},
-      forward?: unknown,
+      { forward = false, route = 'pfm' as 'pfm' | 'cctp' } = {},
     ) => {
       const next = sent.shift();
       if (!next) throw t.fail('nothing sent');
       const { evidence } = next;
+      const { EUD } = decodeAddressHook(evidence.aux.recipientAddress).query;
 
       // C3 - Contract MUST calculate AdvanceAmount by ...
       // Mostly, see unit tests for calculateAdvance, calculateSplit
       const toReceive = forward
         ? { value: evidence.tx.amount }
         : feeTools.calculateAdvance(
             AmountMath.make(USDC, evidence.tx.amount),
-            chainHub.resolveAccountId(evidence.aux.recipientAddress),
+            chainHub.resolveAccountId(EUD as AccountId),
           );
 
       if (forward) {
         t.log(who, 'waits for fallback / forward');
         t.deepEqual(bank, []); // no vbank GIVE / GRAB
       }
 
-      const { EUD } = decodeAddressHook(evidence.aux.recipientAddress).query;
-
+      const transferType = '/ibc.applications.transfer.v1.MsgTransfer';
```

Comment:
I think this would read clearer inline

Reply:
inlining it turned 1 line into 4... but ok... I see a different way to phrase it.